astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -828,6 +828,39 @@ def set_axislabel_visibility_rule(self, rule):
             will only be drawn if tick labels were drawn on that axis.
         """
         self._axislabels.set_visibility_rule(rule)
+    
+    def set_visible(self, visible):
```

Comment:
I think this need a change log under "api"?

Reply:
Yes. Please see https://github.com/astropy/astropy/blob/main/docs/changes/README.rst . Thanks!

Reply:
Also since your pre-commit check is failing, https://docs.astropy.org/en/latest/development/development_details.html#pre-commit

Reply:
@pllim shouldn't new API go in a feature entry?

Reply:
Oh yeah. You might be right. Please change to `18411.feature.rst`. Thanks!